pkg/oc/cli/cmd/newapp.go

```diff
@@ -385,13 +390,42 @@ func (o *NewAppOptions) RunNewApp() error {
 			fmt.Fprintf(out, "%sTrack installation of %s with '%s logs %s'.\n", indent, installing[i].Name, o.BaseName, installing[i].Name)
 		}
 	case len(result.List.Items) > 0:
+		//if we don't find a route we give a message to expose it
+		containRoute := false
+		for _, i := range result.List.Items {
+			switch i.(type) {
+			case *routeapi.Route:
+				containRoute = true
```

Comment:
I meant to push this up to around line 377 to save the additional iteration - is that possible?